File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/UtbetalingsgradUT.java
package no.nav.domain.pensjon.regler.beregning2011;

import java.io.Serializable;

/**
 * @author Steinar Hjellvik (Decisive) - PK-6458
 */
public class UtbetalingsgradUT implements Serializable {


    /**
     * Året utbetalingsgraden gjelder for.
     */
    private int ar;

    /**
     * Utbetalingsgraden hentes fra uforetrygdOrdiner.avkortingsInformasjon.utbetalingsgrad.
     */
    private int utbetalingsgrad;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/UtbetalingsgradUT.kt
package no.nav.pensjon.regler.domain.beregning2011

import java.io.Serializable

class UtbetalingsgradUT : Serializable {
    /**
     * året utbetalingsgraden gjelder for.
     */
    @JvmField
    var ar = 0

    /**
     * Utbetalingsgraden hentes fra uforetrygdOrdiner.avkortingsInformasjon.utbetalingsgrad.
     */
    var utbetalingsgrad = 0

}